src/ExpressionEvaluator/Core/Source/ExpressionCompiler/CustomTypeInfo.cs

```diff
@@ -1,37 +1,196 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.CodeAnalysis.Collections;
+using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
 using System;
 using System.Collections.ObjectModel;
-using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
+using System.Text;
 
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
-    /// <remarks>
-    /// We can't instantiate <see cref="DkmClrCustomTypeInfo"/> in the unit tests
-    /// (since we run against a reference assembly), so we use this type as an
-    /// intermediary.
-    /// </remarks>
-    internal struct CustomTypeInfo
+    internal static class CustomTypeInfo
```

Comment:
Never mind, I just noticed the old comment. Maybe that comment should stay.

---
In reply to: [77420605](https://github.com/dotnet/roslyn/pull/13540#discussion_r77420605) [](ancestors = 77420605)